rules/windows/credential_access_dump_registry_hives.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to export a registry hive which may contain credentials using the Windows reg.exe tool.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+max_signals = 33
```

Comment:
Do you think this will be noisy, or should we just do the default max_signals?
```suggestion

```

Reply:
so keeping max signals for?

Reply:
deleted